src/main/java/me/justeli/coins/handler/DropHandler.java

```diff
@@ -283,6 +297,18 @@ private void drop (int amount, @Nullable Player player, @NotNull Location locati
         }
     }
 
+    private double getLevelledMobsMultipliedAmount(double increment){
```

Comment:
could add a `LivingEntity` parameter instead of getting it through `this.deadMob` (when applying the requested in other comment)

(and yeah as said before i'd rather have this function moved to the hook)

Reply:
I changed it